src/server.py

```diff
@@ -507,19 +473,18 @@ def run_jobnames_query():
 @json_response
 def run_dailyjob_query():
     start_date, end_date = clean_date_params(request.args)
-    db = create_db_connnection()
-    cursor = db.cursor()
-    query = "select date, platform, branch, numpushes, numjobs, sumduration from dailyjobs \
-             where date>='%s' and date <='%s'\
-             order by case platform \
-                when 'linux' then 1 \
-                when 'osx' then 2  \
-                when 'win' then 3  \
-                when 'android' then 4 \
-                end" % (start_date, end_date)
-    cursor.execute(query)
+
+    data = session.query(Dailyjobs.date, Dailyjobs.platform, Dailyjobs.branch, Dailyjobs.numjobs,
+                         Dailyjobs.sumduration).\
+        filter(Dailyjobs.date.between(start_date, end_date)).order_by(case([
+            (Dailyjobs.platform == 'linux', 1),
+            (Dailyjobs.platform == 'osx', 2),
+            (Dailyjobs.platform == 'win', 3),
+            (Dailyjobs.platform == 'android', 4)],
```

Comment:
as a note, when we don't sanitize the platforms coming in, we break things like this